### PR TITLE
add pkg.info to Pkg module

### PR DESCRIPTION
**Pull request by lucidd**
This i work in progress and not ready to be merged yet since attribute names might change upstream.

### CODE REVIEW

File: src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java [hunk 1]
```diff
@@ -43,6 +45,119 @@ public String getSummary() {
         }
     }
 
+    /**
+     * Information about a package as returned by pkg.info_installed and
+     * pkg.info_available
+     */
+    public static class Info {
```
Comment (renner) [inline src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java]: Are you sure this should be named just `Info`? Wouldn't something like `PackageInfo` make it more clear what this is?

File: src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java [hunk 2]
```diff
@@ -268,4 +315,28 @@ public StartTime read(JsonReader jsonReader) throws IOException {
             return new StartTime(subStr);
         }
     }
+
+
+    /**
+     *
+     */
```
Comment (renner) [inline src/main/java/com/suse/saltstack/netapi/parser/JsonParser.java]: Please either remove this empty comment or add some documentation to this class.

**Comment by renner** [comment]
Looks good to me! :+1: Feel free to merge as soon as you are ready.

File: src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java (hunk 1 above)
Comment (renner) [inline src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java]: Nevermind, I see now that it should be used as `Pkg.Info`, makes sense.

Comment (lucidd) [inline src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java]: The idea here is similar to what we have with `Key` that the module name is part of the path you specify when referring to the type.

```java
import com.suse.saltstack.netapi.calls.modules.Pkg;

Pkg.Info a = ...
```

**Pull request merged**